Add JSON schema and example data for imagery, Introduced a new folder structure to improve maintainability

iamrajeshk opened the pull request:
This pull request introduces a new JSON schema and an example dataset for managing imagery source metadata. The changes include defining a schema for validating imagery source data and providing a sample JSON file that adheres to this schema.

### Schema Definition for Imagery Sources:
* Added a new JSON schema in `documentation/imagery/schema.json` to define the structure and validation rules for imagery source metadata. This includes fields for attribution, description, extent (with `max_zoom` and polygon coordinates), icon, ID, name, type (e.g., `tms`, `wmts`, `xyz`), and URL. The schema enforces required fields and data types.

### Example Imagery Source Data:
* Added a sample JSON file in `documentation/imagery/example.json` that adheres to the new schema. This example describes a Hexagon Imagery source for Washington state, including metadata such as attribution, description, geographic extent, and WMTS URL.… structure to improve maintainability

uw-amy-bordenave commented:
Looks good to me, thanks @iamrajeshk.
I made some additional minor edits to fix a typo, update the directory structure, and apply the Prettier formatter.
Requesting review from @cyrossignol and/or @wisamuw before merge :)

cyrossignol commented on documentation/imagery/example.json:
```diff
@@ -0,0 +1,102 @@
+[
```
I know this is just an example--we may want to scrub the real URL, or generalize the whole example. I don't think our license allows us to distribute the imagery in this way.

iamrajeshk commented:
> I updated the imagery example to mirror the suggestions from the [OSMF License Attribution Guidelines](https://osmfoundation.org/wiki/Licence/Attribution_Guidelines).
> 
> Also note that these are not vector tiles so I've removed that text - OSM vector tiles are provided at `https://vector.openstreetmap.org/shortbread_v1/{z}/{x}/{y}.mvt` ([OSM Forum: Vector tiles on OSMF hardware](https://community.openstreetmap.org/t/vector-tiles-on-osmf-hardware/121501))
> 
> @iamrajeshk Let me know of any objections; otherwise, this is ready to merge pending your confirmation.

Looks good @uw-amy-bordenave. Please go ahead.
